docs/markdown/Using Pants/concepts/goals.md

```diff
@@ -20,71 +20,58 @@ You'll see more goals activated as you activate more [backends](doc:enabling-bac
 
 For example:
 
-```bash
-❯ ./pants test project/app_test.py
-15:40:37.89 [INFO] Completed: test - project/app_test.py:tests succeeded.
-
-✓ project/app_test.py:tests succeeded.
+```
+❯ ./pants count-loc project/app_test.py
```

Comment:
I changed from `test` to `count-loc` because this is more likely to work in brand new repository.